auth/jwt.go

```diff
@@ -32,6 +35,12 @@ var (
 		MultiTenancyField,
 		"AccountID",
 	}
+
+	// multiCompartmentVariants all possible multi-compartment names
+	multiCompartmentVariants = []string{
+		MultiCompartmentField,
+		"CompartmentID",
+	}
```

Comment:
```suggestion
```

Let's not do this. You have defined the field name as `compartment_id`, there's no reason for it to then attempt to read `CompartmentID`. I have no idea why AccountID supports two variations, probably backwards compatibility to dead code